libs/feature-action-bar/src/lib/action-bar.component.ts

```diff
@@ -0,0 +1,162 @@
+import {
+  state,
+  style,
+  trigger,
+  transition,
+  animate
+} from '@angular/animations';
+import { Component, HostBinding, OnInit } from '@angular/core';
+import { BehaviorSubject, combineLatest } from 'rxjs';
+import { map, shareReplay, tap, distinctUntilChanged } from 'rxjs/operators';
+import { CommandRunner, Command, CommandStatus } from '@angular-console/utils';
+import { ContextualActionBarService } from '@nrwl/angular-console-enterprise-frontend';
+const ACTION_BAR_ANIMATION_NAME = 'actionBarAnimation';
+type ActionBarAnimationStates = 'hidden' | 'showing';
+
+const TERMINAL_PADDING = 44;
+const COMMAND_HEIGHT = 72;
+const ACTION_BAR_HEIGHT = 48;
+
+@Component({
+  selector: 'angular-console-action-bar',
+  templateUrl: './action-bar.component.html',
+  styleUrls: ['./action-bar.component.scss'],
+  animations: [
+    trigger(ACTION_BAR_ANIMATION_NAME, [
+      state('no-actions', style({ height: '0' })),
+      state('action-bar', style({ height: '64px' })),
+      state('action-panel', style({ height: '100vh' }))
+    ]),
+    trigger('growShrink', [
+      state('void', style({ height: 0, opacity: 0 })),
+      state('contract', style({ height: 0, opacity: 0 })),
+      state('*', style({ height: '*', opacity: 1 })),
+      transition(`contract <=> *`, animate(`250ms ease-in-out`)),
+      transition(`:enter`, animate(`250ms ease-in-out`)),
+      transition(`:leave`, animate(`250ms ease-in-out`))
+    ]),
+    trigger('growShrinkTerminal', [
+      state(
+        'void',
+        style({
+          height: 0,
+          opacity: 0,
+          'margin-bottom': '0',
+          'padding-top': '0',
+          'padding-bottom': '0'
+        })
+      ),
+      state(
+        '*',
+        style({
+          height: '{{terminalHeight}}', // use interpolation
+          opacity: 1,
+          'margin-bottom': '16px',
+          'padding-top': '8px',
+          'padding-bottom': '8px'
+        }),
+        { params: { terminalHeight: '0' } }
+      ),
+      transition(`* <=> *`, animate(`250ms ease-in-out`))
+    ])
+  ]
+})
+export class ActionBarComponent {
+  @HostBinding(`@${ACTION_BAR_ANIMATION_NAME}`)
+  actionPanelAnimation: ActionBarAnimationStates = 'hidden';
+
+  CommandStatus = CommandStatus;
```

Comment:
you will need to use a different enum here as the backend returns slightly different values

Reply:
Fixed!